app/metrics.yaml

```diff
@@ -1562,6 +1562,21 @@ collections:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-09-01"
+  total_collections:
+    type: quantity
```

Comment:
The docs say,

> Note: Quantities are currently only allowed for GeckoView metrics (the gecko_datapoint parameter is present) and thus have only a Kotlin API.

Did you verify with the glean team that this was okay?

Reply:
Ah I figured that was a note as to why api is not available in other languages.  I will double check it the Glean team

Reply:
Yup, @mcomella instinct is correct :-D This should not be used!

Reply:
I will change it up.